script/swap_and_bridge/test/integration_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../../../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+if [ -z "$L1_TOKEN_ADDR_LIDO" ]
+then
+    echo "Please set L1_TOKEN_ADDR_LIDO in .env file"
+    exit 1
+fi
+
+if [ -z "$L2_TOKEN_ADDR_LIDO" ]
+then
+    echo "Please set L2_TOKEN_ADDR_LIDO in .env file"
+    # Unlike Diva, L2_TOKEN_ADDR_LIDO cannot be deployed as the bridge is setup to work only with a predefined L2 token.  
+    exit 1
+fi
+
+echo "Running integration tests for Lido contract..."
+forge script -vv --rpc-url "http://localhost:8545" script/swap_and_bridge/contracts/SwapAndBridge_integration_test.s.sol:TestIntegrationScript $L1_LIDO_BRIDGE_ADDR $L1_TOKEN_ADDR_LIDO $L2_TOKEN_ADDR_LIDO --sig 'run(address,address,address)'
+echo "Done."
+
+if [ -z "$L1_TOKEN_ADDR_DIVA" ]
+then
+    echo "Please set L1_TOKEN_ADDR_DIVA in .env file"
+    exit 1
+fi
+
+if [ -z "$L2_TOKEN_ADDR_DIVA" ]
```

Comment:
That this works properly `L2_TOKEN_ADDR_DIVA` need to be set inside `.env` files (example, mainnet) like this:
`L2_TOKEN_ADDR_DIVA=`
and not like it currently is:
`L2_TOKEN_ADDR_DIVA=0x`

If I put `L2_TOKEN_ADDR_DIVA=` inside my `.env` file I get following error:
```
L2_TOKEN_ADDR_DIVA variable inside .env file is not set.
Deploying 'Wrapped Diva Ether Token' on L2...
Error: 
server returned an error response: error code 3: execution reverted
```

Reply:
fixed